content-src/experiments/send.yaml

```diff
@@ -0,0 +1,56 @@
+id: 13
+title: 'Send'
+slug: send
+platforms: ['web']
+web_url: 'https://send.firefox.com'
+thumbnail: /static/images/experiments/send/experiments_experiment/thumbnail.png
+description: >
+  Send files through a safe, private, and encrypted link that automatically expires to ensure your stuff does not remain online forever.
+introduction: >
+  TKTKTKTK
+image_twitter: /static/images/experiments/send/social/mv-twitter.jpg
+image_facebook: /static/images/experiments/send/social/mv-facebook.jpg
+changelog_url: 'https://github.com/mozilla/send/master/CHANGELOG.md'
+contribute_url: 'https://github.com/mozilla/send'
+bug_report_url: 'https://github.com/mozilla/send/issues'
+discourse_url: 'https://discourse.mozilla-community.org/c/test-pilot/send'
+privacy_notice_url: 'https://github.com/mozilla/send/blob/master/docs/metrics.md'
+measurements:
+  - >
+   TKTKTKTK
+gradient_start: '#32CADD'
+gradient_stop: '#3199FF'
+details:
+  -
+    image: /static/images/experiments/send/experiments_experimentdetail/detail1.jpg
+    copy: >
+      Access Min Vid from YouTube, Vimeo, and SoundCloud players.
+contributors:
+  -
+    display_name: 'Abhinav Adduri'
+    title: 'Engineering Intern'
+    avatar: /static/images/experiments/min-vid/avatars/dave-justice.jpg
```

Comment:
Bonus points if we can just use the placeholder instead of pointing to min-vid avatars:
https://github.com/mozilla/testpilot/blob/master/frontend/src/images/experiments/avatars/generic-profile.png